.husky/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:lts
+WORKDIR /app
+COPY package.json /app
+RUN npm install
+COPY . /app
+CMD ["npm", "start"]
```

Comment:
Any reason the Dockerfile is in the `.husky` folder? ^^

(I'm not super familiar with NodeJS development, maybe that's actually common practice and I'm just unaware of it)

Reply:
Woops sorry about that! I don't know how it landed there but it has been fixed!